ibc-core/ics03-connection/src/handler/mod.rs

```diff
@@ -1,4 +1,50 @@
+use ibc_core_client::types::error::ClientError;
+use ibc_core_handler_types::error::ContextError;
+use ibc_core_host::types::identifiers::ClientId;
+use ibc_primitives::proto::Any;
+
 pub mod conn_open_ack;
 pub mod conn_open_confirm;
 pub mod conn_open_init;
 pub mod conn_open_try;
+
+pub(crate) fn unpack_host_client_state<CS>(
```

Comment:
Could we add a doc string for this method?

Reply:
ebd2327